docs/xxignoreunrecognizedrestoreoptions.md

```diff
@@ -0,0 +1,52 @@
+<!--
+* Copyright (c) 2017, 2023 IBM Corp. and others
+*
+* This program and the accompanying materials are made
+* available under the terms of the Eclipse Public License 2.0
+* which accompanies this distribution and is available at
+* https://www.eclipse.org/legal/epl-2.0/ or the Apache
+* License, Version 2.0 which accompanies this distribution and
+* is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* This Source Code may also be made available under the
+* following Secondary Licenses when the conditions for such
+* availability set forth in the Eclipse Public License, v. 2.0
+* are satisfied: GNU General Public License, version 2 with
+* the GNU Classpath Exception [1] and GNU General Public
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] https://openjdk.org/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# -XX:\[+|-\]IgnoreUnrecognizedRestoreOptions
+
+**(Linux&reg; x86, Linux on POWER LE&reg;, and Linux on IBM Z&reg; only)**
+
+This option specifies whether the VM ignores any unrecognized options at the time of restore.
+
+:fontawesome-solid-triangle-exclamation:{: .warn aria-hidden="true"} **Restrictions:** This option takes effect only when `-XX:+EnableCRIUSupport` is enabled. Warning messages for unrecognized or unsupported options are displayed. This option can be used only on restore.
+
+
+## Syntax
+
+        -XX:[+|-]IgnoreUnrecognizedRestoreOptions
+
+| Setting               | Effect  | Default                                                                            |
+|-----------------------|---------|:----------------------------------------------------------------------------------:|
+| `-XX:+IgnoreUnrecognizedRestoreOptions` | Enable  |                                                                                 |
+| `-XX:-IgnoreUnrecognizedRestoreOptions` | Disable | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span>  |
+
+
+## Explanation
+
+The VM enables the addition of VM options on restore through the `CRIUSupport.registerRestoreEnvVariables` and the `CRIUSupport.registerRestoreOptionsFile` APIs. The VM fails to restore if any of the options that are provided are not supported or do not exist and throws the `JVMRestoreException` message. The `-XX:+IgnoreUnrecognizedRestoreOptions` option disables the failure and allows the VM to continue even if some options are not used.
```

Comment:
Think the use of "restore" in most cases in these docs needs changing. Then there's the usual comment about passive voice. This instance could be "When you restore a checkpoint image, you can specify additional command-line options by using the..."

`CRIUSupport.registerRestoreEnvVariables` is for specifying environment variables not command-line options. Although the API mentions the `OPENJ9_RESTORE_JAVA_OPTIONS` env var, which can be used to specify new command-line options, if that's what this text is referring to. That env var doesn't appear to be mentioned in these docs but it probably should be somewhere.

"The VM fails to restore..." could be (assuming you change the previous sentence) "If you specify options that do not exist, or are unsupported, the VM by default fails to start and throws a `JVMRestoreException` error."
(JVMRestoreException is a Java exception not a message, as you can see in the API doc at https://eclipse-openj9.github.io/openj9-docs/api/jdk11/openj9.criu/org/eclipse/openj9/criu/JVMRestoreException.html. Technically, we should call it a `JVMRestoreException` exception, which is good for any future translation but looks odd in English)